### DISCUSSION TITLE
怎么才能让 “NFT Market” 字样显示在页面中间，而 <ConnectButton /> 显示在页面最右边？

### DISCUSSION THREAD

**Discussion by linghuccc** [openbuildxyz/Web3-Frontend-Bootcamp#662] [Q&A]
我尝试了几种方法，最接近的就是下面的方法：
```
		<div className="bg-gray-700 py-8 flex justify-between items-center">
			<div className="text-white text-3xl mx-auto">NFT Market</div>
			<div className="mr-4">
				<ConnectButton accountStatus="address" showBalance={true} />
			</div>
		</div>
```

但这也没有达到我想要的效果，因为 "NFT Market" 字样是在 `<div className="text-white text-3xl mx-auto">` 的中间，而我想要的是他在 `<div className="bg-gray-700 py-8 flex justify-between items-center">` 的中间。

求教各位前端大神，有什么办法？

**Comment by haobarry** [comment]
你这种上面都写了 justify-between这个里就不好弄 了  这个可以去掉，然后自己用margin-left 调整，或者用绝对定位居中
